core-services/sync-rover-groups/_config.yaml

```diff
@@ -59,6 +59,7 @@ groups:
     - hosted-mgmt
   multiarch-devel:
     clusters:
+    - app.ci
```

Comment:
This is for the human user. I guess you also need https://docs.ci.openshift.org/docs/how-tos/use-registries-in-build-farm/#token-for-programmatic-access-to-qci